index.html

```diff
@@ -20,14 +20,19 @@
             <h1>Match Mayden Heaven</h1>
             <h1 class="questionMark"><a href="#footer" title="Click to view game instructions">&#63</a></h1>
         </header>
-        <div class="sideBox">
+        <div class="sideBox sideBox1">
             <h3>Turns:</h3>
             <div id="turnContainer">0</div>
         </div>
+        <div class="sideBox sideBox2">
+            <h3>Time:</h3>
+            <div id="timeContainer">00:00</div>
+        </div>
         <main>
             <div id="winScreen">
                 <div class="youWin">You Win!</div>
                 <p>You finished in <span id="totalTurns">0</span> turns</p>
+                <p>and it took you <span id="totalTime">0</span></p>
```

Comment:
Format the default 0 time to '00:00' (although no-one should see this, it's consistent!)